client/packages/common/src/ui/components/navigation/Tabs/DetailTabs.tsx

```diff
@@ -42,17 +41,19 @@ export const DetailTabs: FC<DetailTabsProps> = ({
   const currentTab = isValidTab(currentUrlTab)
     ? currentUrlTab
     : tabs[0]?.value ?? '';
+  const { showConfirmation } = useConfirmOnLeaving(false);
 
   // Inelegant hack to force the "Underline" indicator for the currently active
   // tab to re-render in the correct position when one of the side "drawers" is
   // expanded. See issue #777 for more detail.
   const { isOpen: detailPanelOpen } = useDetailPanelStore();
   const { isOpen: drawerOpen } = useDrawer();
-  const { showConfirmation } = useConfirmOnLeaving(false);
-  const handleResize = useCallback(
-    () => debounce(() => window.dispatchEvent(new Event('resize')), 100),
-    []
-  );
+  const handleResize = useCallback(() => {
+    window.dispatchEvent(new Event('resize'));
+  }, []);
+  useEffect(() => {
+    handleResize();
+  }, [detailPanelOpen, drawerOpen, handleResize]);
```

Comment:
This doesn't work for me, the bar is not centerred correctly...

```ts
useEffect(() => {
    setTimeout(() => {
      window.dispatchEvent(new Event('resize'));
    }, 100);
  }, [detailPanelOpen, drawerOpen]);
```
This works better though. Don't ask me why...

@mark-prins where is that underline drawn? is that in MUI or are we doing it?

Reply:
But work the same for me... I see no difference in the underline movement

Reply:
Without the debounce or the delay the event seem to be created to early, i.e. while the sidebar is still expanding/shrinking. Thus the underline stops to early...

Reply:
Just a guess: there are multiple debounce calls and the time of the first debounce call + the delay is still before the animation finished, thus it doesn't work for me. My machine is slower than yours, so I can see this and you can't (?).